packages/itwinui-css/src/input/searchbox.scss

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin iui-expandable-searchbox {
+  width: var(--_iui-input-min-height);
+  transition: width var(--iui-duration-3) ease;
+  isolation: isolate;
+  overflow: hidden;
+
+  &:has(.iui-searchbox-open-button:focus-visible) {
```

Comment:
needs fallback for firefox

Reply:
Added fallback. Seems a bit weird behaviour with ::focus-within styles it gets from flex container.